docs/troubleshooting/rancher_prerequisite.md

```diff
@@ -0,0 +1,50 @@
+# Prerequisites to run CStor-CSI in rancher based clusters
+
+## Intro
+CStor-CSI provides block volume support through the iSCSI protocol. Therefore, the iSCSI client(initiator) presence on all Kubernetes nodes is required.
+
+## Step to be performed on Rancher based K8s cluster
+
+- If you are using RancherOS as the operating system, you need to enable the iSCSI service and start iSCSI service on all the worker nodes.
+- If you are using Ubuntu or RHEL as the operating system, you need to
+    - Verify if iSCSI initiators are installed on all nodes.
```

Comment:
in ubuntu and rhel also, iscsi initiator would not be there by default. we need to install if it is not there